ui/lib/client/translations/zh.yaml

```diff
@@ -7,6 +7,7 @@ error:
     user:
       signin:
         invalid_code: 授权码无效或已过期
+      lack_required_privileges: 此用户缺少访问 TiDB Dashboard 所需的权限
```

Comment:
I think we should provide some better guidelines for what kind of privileges user need. Maybe better to leave a TODO and add a link to the documentation later.

Reply:
Yep, @crazycs520 has discussed it with me, we should explain what privileges it needs in the doc.

Reply:
Done.